tidb.go

```diff
@@ -88,7 +89,8 @@ var (
 	}
 	stores = make(map[string]kv.Driver)
 	// Debug is the switch for pprof. If Debug is true, tidb will start with pprof on.
-	Debug = true
+	Debug       = true
+	EnablePprof = os.Getenv("TIDB_PPROF") != "0"
 	// PprofAddr is the pprof url.
 	PprofAddr = "localhost:8888"
```

Comment:
As i mentioned in #684, we will use configuration to handle this.
Right now, let's just handle the flag.
@Unknwon CI failed, please add comment to pass go lint.  https://travis-ci.org/pingcap/tidb/builds/95520320